### ISSUE TITLE
Rule proposal: Test function bodies should always be blocks

### ISSUE THREAD

**Issue by raphinesse** [tlvince/eslint-plugin-jasmine#316]
Would there be interest in a rule that enforces that test function bodies always be blocks?

```js
// BAD: Test function bodies should always be blocks (at 1:38)
it('should always use blocks', () => testSubject().then(
// ----------------------------------^
  result => expect(result).toBe(42)
));

// OK
it('should always use blocks', () => {
  return testSubject()
    .then(result => expect(result).toBe(42));
});
```

[Implementation prototype with samples](https://astexplorer.net/#/gist/756c9eddeac447fd247a0bf8fdedbf40/c56cc7430493ddb01183cd80fd7a4681b9e68210)

**Comment by DianaSuvorova** [comment]
Yes! New rules are welcome! 

I wouldn't include this one as an error for default configuration, but definitely would be a good contribution so people can decide if they want this as a convention for their projects.
